zipline/api.py

```diff
@@ -53,5 +54,6 @@
     'execution',
     'math_utils',
     'slippage',
-    'time_rules'
+    'time_rules',
+    'calendars',
```

Comment:
I didn't recall discussing this addition. @ssanderson This align with your understanding?